.github/workflows/nightly-builds.yml

```diff
@@ -178,8 +178,8 @@ jobs:
           sudo apt-get install graphviz
           # genjavadoc does not support Scala 3.x
           sbt \
-            -Dpekko.genjavadoc.enabled=${{ matrix.scalaVersion != '3.3' }} \
-            "+~ ${{ matrix.scalaVersion }} doc"
+            -Dpekko.genjavadoc.enabled=${{ matrix.scalaVersion.startsWith("3") }} \
```

Comment:
https://github.com/apache/pekko/pull/1324 changed the matrix scalaVersions to be of the form `3.3.x`. I think it is safer not to rely exact matches here.

Reply:
`startsWith(matrix.scalaVersion, "3") ` ?
https://docs.github.com/en/actions/learn-github-actions/expressions#startswith